src/App.tsx

```diff
@@ -15,10 +22,16 @@ function App() {
   return (
     <FoodprintContextProvider>
       <div className="App">
-        <IngredientsContainer />
-        <Recipes />
-        <header>It is time to scream, my dudes</header>
-        {openPopup && <LandingPagePopup toggle={handleClose}/>}
+        <Routes>
```

Comment:
if we are doing routes wee will need to separate the routes from the component they  are rendering so this page isn't clogged up